manifests/c/CosmicPredator/Chibi/1.0.1/CosmicPredator.Chibi.installer.yaml

```
@@ -1,12 +1,12 @@
-# Created using wingetcreate 1.9.2.0
+# Created with Sundry.
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
 
 PackageIdentifier: CosmicPredator.Chibi
 PackageVersion: 1.0.1
 InstallerType: inno
 Installers:
 - Architecture: x64
-  InstallerUrl: https://github.com/CosmicPredator/chibi-cli/releases/latest/download/chibi_installer.exe
+  InstallerUrl: https://github.com/CosmicPredator/chibi-cli/releases/download/1.0.1/chibi_1.0.1_windows_amd64.exe
   InstallerSha256: 611D970B3B3B8FF3395D1E05837371977AF89C12227904EE736B98058571BCAD
```

```suggestion
  InstallerSha256: AF75B683E9BD86D544B4AC60101BC551DC6EA2E91672DBF03D67C57E1A5A2923
```

Hash by https://dev.azure.com/shine-oss/winget-pkgs/_build/results?buildId=88352&view=logs&j=68997d86-8ece-5ae8-3ac5-ccdbfa458731&t=a6d88639-1ba6-5d70-db6d-8e283a18f2b4&l=37